src/diffusers/pipelines/alt_diffusion/pipeline_alt_diffusion.py

```diff
@@ -471,6 +471,18 @@ def __call__(
                 The frequency at which the `callback` function will be called. If not specified, the callback will be
                 called at every step.
 
+        Examples:
+        ```py
+        >>> import torch
+        >>> from diffusers import AltDiffusionPipeline
+
+        >>> pipe = AltDiffusionPipeline.from_pretrained("runwayml/stable-diffusion-v1-5", torch_dtype=torch.float16)
```

Comment:
```suggestion
        >>> pipe = AltDiffusionPipeline.from_pretrained("BAAI/AltDiffusion-m9", torch_dtype=torch.float16)
```

Reply:
Let's try to use the official checkpoint here :-) 
https://huggingface.co/BAAI/AltDiffusion-m9